oidc-api/src/main/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandler.java

```diff
@@ -322,6 +323,19 @@ public APIGatewayProxyResponseEvent authoriseRequestHandler(
             }
         }
 
+        if (!client.getIsActive()) {
+            String errorMsg = "Client configured as not active in Client Registry";
+            LOG.error(errorMsg);
+            LOG.warn("Redirecting");
```

Comment:
Not sure we need these extra lines. They don't give any additional information. I know they were copied from elsewhere, but still...